docs/development/baremetal_setup.md

```diff
@@ -0,0 +1,66 @@
+# Bare metal machine setup with TheRock
+
+## Linux
```

Comment:
I don't see much in here that is "bare metal" specific, which is good. Maybe we can do some other documentation restructuring to explain how flexible the installation is, reducing dependence on containers / virtual machines / etc. That could also help with the duplication between other documentation files.

As for Windows, the "bare metal" concept doesn't usually apply as much as it does on Linux servers, but yes, we should have a "installing on Windows" document separate from "building from source on Windows"... when we have complete releases available on Windows.